R/ducktbl.R

```diff
@@ -5,22 +5,59 @@
 #' in addition to the default classes.
 #' This ensures that dplyr methods are dispatched correctly.
 #' For such objects,
-#' dplyr verbs such as [mutate()], [select()] or [filter()]  will attempt to use DuckDB.
-#' If this is not possible, the original dplyr implementation is used.
+#' dplyr verbs such as [mutate()], [select()] or [filter()]  will use DuckDB.
 #'
-#' `duck_tbl()` works like [tibble()].
-#' In contrast to dbplyr, duckplyr data frames are "eager" by default.
-#' To avoid unwanted expensive computation, they can be converted to "lazy" duckplyr frames
-#' on which [collect()] needs to be called explicitly.
+#' `duck_tbl()` works like [tibble()], returning an "eager" duckplyr data frame by default.
+#' See the "Eager and lazy" section below.
 #'
-#' @details
-#' Set the `DUCKPLYR_FALLBACK_INFO` and `DUCKPLYR_FORCE` environment variables
-#' for more control over the behavior, see [config] for more details.
+#' @section Eager and lazy:
+#' Data frames backed by duckplyr behave as regular data frames in almost all respects.
+#' In particular, direct column access like `df$x`,
+#' or retrieving the number of rows with [nrow()], works identically.
+#' For a duckplyr frame that is the result of a dplyr operation,
```

Comment:
In that sentence the information that the computation is not carried out immediately, is missing.